source/jsonnet/england-wales/individual/blocks/employment/armed_forces.jsonnet

```diff
@@ -53,23 +49,42 @@ local question(title) = {
 };
 
 local nonProxyTitle = 'Have you previously served in the UK Armed Forces?';
+local nonProxyGuidanceHeader = 'Why your answer is important';
+local nonProxyGuidanceContent = [
+  {
+    description: 'Your answer will help your local community by providing information needed to support people who used to serve in the UK Armed Forces but have now left.',
+  },
+  {
+    description: 'Councils and government will use this information to carry out the commitments they made under the Armed Forces Covenant. This is a promise by the nation to ensure that those who serve or who have served in the UK Armed Forces, and their families, are not disadvantaged.',
```

Comment:
We should assign this string to a variable and reuse it as it is duplicated below. Will also avoid us having to update it in two places if this were to change again.